src/components/LoginPanel.tsx

```diff
@@ -9,13 +9,35 @@ import styled, { useTheme } from "styled-components";
 
 const MiddleDiv = styled.div`
     max-width: 400px;
-    border: solid 1px ${(props) => props.theme.foregroundColor};
+    box-shadow: 0 0 15px ${(props) => props.theme.shadow};
     border-radius: 10px;
     padding: 50px;
     margin: auto;
+    background-color: ${(props) => props.theme.main.background[0]};
 
-    & button:hover {
-        color: ${(props) => props.theme.secondaryForegroundColor};
+    --fonts-buttonFontFamily: Helvetica, Arial, sans-serif;
+    --fonts-bodyFontFamily: Helvetica, Arial, sans-serif;
+    --fonts-inputFontFamily: Helvetica, Arial, sans-serif;
+    --fonts-labelFontFamily: Helvetica, Arial, sans-serif;
+
+    & input {
+        border-color: ${(props) => props.theme.main.background[0]};
+        background-color: ${(props) => props.theme.main.background[1]};
+    }
+
+    // Google Chrome password autofill will automatically make the background blue and the word black, which is inconsistent with our current theme
+    // The below forces the background on Google Chrome to be our desired background
+    // Since Google Chrome used !important in their -webkit-autofill, we cannot override their background-color setting
+    & input:-webkit-autofill,
+    & input:-webkit-autofill:hover,
+    & input:-webkit-autofill:focus,
+    & input:-webkit-autofill:active {
+        -webkit-box-shadow: 0 0 0 9999px ${(props) => props.theme.main.background[2]} inset !important;
+        -webkit-text-fill-color: ${(props) => props.theme.main.foreground[2]} !important;
+    }
+
+    & button {
+        border: transparent;
     }
 `;
 
```

Comment:
I think the <{}> in React.FC can be removed id its not taking any props.

Reply:
Removed